skills/declarative/calendarSkill/dialogs/ListDialog/ListDialog.dialog

```diff
@@ -108,12 +108,18 @@
             }
           ]
         },
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "tdNJ28"
+          }
+        },
         {
           "$kind": "Microsoft.IfCondition",
           "$designer": {
             "id": "P1awzy"
           },
-          "condition": "(dialog.pageIndex * dialog.pageSize) - count(dialog.Items) <= dialog.pageSize",
+          "condition": "dialog.pageIndex * dialog.pageSize <  count(dialog.Items)",
```

Comment:
I wonder if this equation is "too much math gymnasium" for non-technical customers to understand and whether or not we want to simplify this further. 

Basically what we are saying is "given the result set, what is the pages required to display all the information." then there is the question of "given where we are in the page index, are we at the end yet?" If we model it this way would it be easier to understand? And should we do it this way too?

Reply:
@thomas-chung - why don't you take a look at the ListDialog separately and try to model it this way? It'd be a good exercise since that is one of the components we're hoping to ship as a separate reusable asset